### PR TITLE
Update libs

### PR DESCRIPTION
**Pull request by trevorgerhardt**
And ran `mastarm lint --fix`

### CODE REVIEW

**Comment by trevorgerhardt** [comment]
Added a `fix` commit to generate a new release.

File: .travis.yml [hunk 1]
```diff
@@ -2,17 +2,17 @@ sudo: false
 language: node_js
 cache:
   directories:
-    - node_modules
+    - ~/.yarn-cache
```
Comment (evansiroky) [inline .travis.yml]: Let's change this to:

```
cache:
  yarn: true
```

**Review by evansiroky — CHANGES_REQUESTED** [review]
Let's get the travis.yml updated and then merge this.

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
+  yarn: true
 notifications:
   email: false
 node_js:
   - '6'
 before_install:
-  - npm i -g npm@^3.0.0
-before_script:
-  - npm prune
+  - npm i -g yarn
```
Comment (evansiroky) [inline .travis.yml]: I think if you include yarn as the cache strategy that yarn will get installed by travis.

**Comment by trevorgerhardt** [comment]
@mattwigway or @evansiroky could you give this one another review 🙏

File: index.js [hunk 3]
```diff
@@ -12,10 +13,25 @@ import dbg from 'debug'
...
+  height,
+  cutoff,
+  project,
+  interpolation = true
+}: {
+  surface: Uint8Array,
+  width: number,
+  height: number,
+  cutoff: number,
+  project: Function,
```
Comment (evansiroky) [inline index.js]: [The Function type is unsafe and should be avoided](https://flow.org/en/docs/types/functions/#toc-function-type).

You could rewrite as follows:

```js
type CoordinateArray = [number, number]

...
  interpolation = true
}: {
  surface: Uint8Array,
  width: number,
  height: number,
  cutoff: number,
  project: CoordinateArray => CoordinateArray,
  interpolation: boolean
}) {
```

**Review by evansiroky — CHANGES_REQUESTED** [review]
Let's try to not use wildcard types like `Function`, `Object`, `any` or `mixed`.

File: .flowconfig [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[ignore]
```
Comment (evansiroky) [inline .flowconfig:1]: Having a config file like this live in the project seems to diverge from keeping config files in mastarm.  But it does sorta seem like this file must exist in order for flow to run, so maybe it's ok.

File: index.js (hunk 3 above)
Comment (trevorgerhardt) [reply index.js]: oh, I had no idea!

File: .flowconfig (hunk 4 above)
Comment (trevorgerhardt) [reply .flowconfig:1]: Yea, when we run `flow-bin` manually we might be able to avoid this. Not sure tho!

Comment (evansiroky) [reply .flowconfig:1]: I didn't see anything in the [docs](https://flow.org/en/docs/cli/) about providing a cli option for a config file.

File: index.js (hunk 3 above)
Comment (evansiroky) [reply index.js]: I think it may also have the benefit of affecting upstream apps that send typed functions.

**Comment by trevorgerhardt** [comment]
@evansiroky & @mattwigway this is ready for another look too

**Review by evansiroky — APPROVED** [review]
👍

**Review by evansiroky — CHANGES_REQUESTED** [review]
Will approve once code coverage is setup correctly

File: .travis.yml [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
   - '6'
+cache:
+  yarn: true
 before_install:
-  - npm i -g npm@^3.0.0
-before_script:
-  - npm prune
+  - yarn global add codecov
+script:
+  - yarn test -- --coverage
```
Comment (evansiroky) [inline .travis.yml]: You'll need to specify the coverage directory/files in order for the coverage to work properly.

Comment (evansiroky) [reply .travis.yml]: In this case it would be `yarn test -- --coverage --coverage-paths index.js`

**Review by evansiroky — APPROVED** [review]
I pushed a change to get the coverage working

**Pull request merged**